src/pybind/mgr/dashboard/frontend/src/app/ceph/new-dashboard/dashboard/dashboard.component.html

```diff
@@ -88,14 +88,237 @@
       </ng-container>
     </cd-card>
   </div>
-
+  <!-- Inventory Card -->
   <div class="row mx-0">
     <cd-card title="Inventory"
              i18n-title
              class="col-sm-3 px-3">
-      <div class="ms-4 me-4">
-        Text
-      </div>
+      <!-- Hosts -->
+      <li class="border-top list-group-item">
+        <div *ngIf="healthData.hosts != null"
+             class="inventory-item d-flex pl-1"
+             id="hosts-item">
+          <div class="ms-2 me-auto"
+               i18n>
+            <a routerLink="/hosts">
+              {{ healthData.hosts }} Hosts</a>
+          </div>
+          <span>
+            {{ healthData.hosts }}
+            <i class="text-success"
+               [ngClass]="[icons.success]"></i>
+          </span>
+        </div>
+      </li>
```

Comment:
I see a lot of HTML code to basically repeat the same row structure:
`<Number> Item - <Number> Ok, <Number> Warning, <Numer> Error`.

Why not creating an `cdInventoryRow` component that basically does this?:
```html
<cdInventoryRow
    i18n-item="Hosts"
    link="/hosts"
    [count-ok]="healthData.hosts"
    [count-warning]=".."
></cdInventoryRow>
```
Or even more compact:
```html
<cdInventoryRow
    i18n-item="Hosts"
    link="/hosts"
    [count]="{ok: healthData.hosts, warn: healthData.hostsWarning}"
></cdInventoryRow>
```
Maybe we don't need to pass a `total` field since it'd be the sum of the individual statuses?

Reply:
Fixed.